### PR TITLE
Ignore deprecated methods of `pydantic.main` in pydocstyle check

### PR DESCRIPTION
**Pull request by hramezani**
Selected Reviewer: @dmontagu

### CODE REVIEW

**Comment by hramezani** [comment]
Please review

File: pydantic/main.py [hunk 1]
```diff
@@ -929,7 +930,7 @@ def parse_file(
     #     "The `from_orm` method is deprecated; set "
```
Comment (Kludex) [inline pydantic/main.py:930]: Since you are here, can you uncomment this? 🤔

Comment (Kludex) [reply pydantic/main.py:930]: I think it was commented by mistake? 🤔

Comment (hramezani) [reply pydantic/main.py]: probably it's not commented by mistake. it could be because of mypy error

Comment (hramezani) [reply pydantic/main.py]: Take a look at your commit message here https://github.com/pydantic/pydantic/pull/5646/commits/32c468c4de12d1d85926987f0ab591370a397023

Comment (Kludex) [reply pydantic/main.py]: Why only on `orm_mode`? 🤔

**Review by Kludex — APPROVED** [review]

File: pydantic/main.py (hunk 1 above)
Comment (hramezani) [reply pydantic/main.py:930]: I will take a look later. Now, I've reverted the change.

**Pull request merged**